docker/index.js

```diff
@@ -50,24 +50,28 @@ app.post('/trigger', async (req, res) => {
 	cache.set(taskId, true, 60);
 
 	// for catch error
-	let uploadMetadataRes; 
+	let uploadMetadataRes;
 	let uploadImgRes;
 
 	const inputPath = transaction.tx_body.operation.ref;
 	const parsedInputPath = parsePath(inputPath);
 
 	// pre-check the output path
-	const rootPath = [...parsedInputPath.slice(0, parsedInputPath.length - 1)]
-	
+	const rootPath = [...parsedInputPath.slice(0, parsedInputPath.length - 1)];
```

Comment:
`[...parsedInputPath.slice(0, parsedInputPath.length - 1)]`
-> `parsedInputPath.slice(0, parsedInputPath.length - 1)`
maybe It works same !

Reply:
Oh ! Thank you :) it was fixed